### ISSUE TITLE
pluginFolder can't be changed

### ISSUE THREAD

**Issue by derkrasseleo** [studiorack/studiorack-app#10]
I set the following directory as my plugin directory:
![image](https://user-images.githubusercontent.com/16163571/167154882-aad0410b-baeb-4f9f-aa4f-ec48e654d821.png)

Still, studiorack puts all plugins into `C:\Program Files (x86)\Common Files\`.

When I try to set the path in the cli I don't get an error but the path also doesn't change:

![image](https://user-images.githubusercontent.com/16163571/167155483-994cf182-78fa-4212-baef-86b2f0d60b9e.png)

**Comment by derkrasseleo** [comment]
This is probably related to cli, that's why I opened an issue in its repository: https://github.com/studiorack/studiorack-cli/issues/3

**Comment by kmturley** [comment]
Found the issue. I had two configs:

1) Core library/cli:
`/Users/username/Library/Preferences/studiorack/config.json`

2) Electron app:
`/Users/username/Library/Application Support/studiorack/config.json`

When using the Electron App, it was loading the first time from 1), but then saving/retrieving from 2) afterwards. Then when running the installation, it would run the CLI tool inside Electron as a sub-process which would read for 1)

I have removed the Electron-specific config 2) and default to the shared config 1)
In addition I don't run the cli as a sub-process, as that was running different versions of code which made debugging difficult!

This commit fixes this issue:
https://github.com/studiorack/studiorack-app/commit/5a7280b20108f094e801ec83fced0c68c6e66ee5
